modules/claims_api/app/controllers/claims_api/v1/forms/disability_compensation_controller.rb

```diff
@@ -17,8 +17,8 @@ class DisabilityCompensationController < BaseDisabilityCompensationController
         before_action { permit_scopes %w[claim.write] }
         before_action :validate_json_schema, only: %i[submit_form_526 validate_form_526]
         before_action :validate_initial_claim, only: %i[submit_form_526 validate_form_526]
-        before_action :validate_documents_content_type, only: %i[upload_supporting_documents]
-        before_action :validate_documents_page_size, only: %i[upload_supporting_documents]
+        # before_action :validate_documents_content_type, only: %i[upload_supporting_documents]
```

Comment:
Include a TODO above it about what needs to happen here, also make sure to include this validation action where the PUT method is also importing a document so that we know to tackle both when the time comes.